core/src/lib.rs

```diff
@@ -814,6 +816,18 @@ pub fn get_last_synced(config: &Config) -> Result<i64, Error<GetLastSyncedError>
     }
 }
 
+pub fn calculate_last_synced(config: &Config) -> Result<String, Error<GetLastSyncedError>> {
+    let last_synced = get_last_synced(config)?;
+
+    if last_synced != 0 {
+        let duration = Duration::milliseconds(DefaultClock::get_time() - last_synced);
+        println!("{}, {}", DefaultClock::get_time(), last_synced);
```

Comment:
no `println` s in core, only log messages. probably should just delete this line